### DISCUSSION TITLE
When are you projecting final release of 3.0?

### DISCUSSION THREAD

**Discussion by adfrancis** [GNS3/gns3-gui#3416] [Q&A] [answered]
Im loving 3.0.. its hitting all the right features im looking for. Looking for to the stable release, any ideas when that's projecting to be?

**Comment by grossmj** [comment]
Hopefully sometime in Q1 or Q2 2023. By curiosity, what features are you looking for?

**Reply by SonarSaied** [reply]
Any update on the timeline?

**Reply by ghost** [reply]
@grossmj created a new blog post: <https://gns3.com/community/blog/whats-new-in-gns3-version-3-0>

There he announced a new release date:

> When can I expect the final release of version 3.0?
>
> A final release is expected during the first quarter of 2024. We will give a better estimate after the first few beta releases. These releases help us to identify major issues that could take some time to fix.

**Comment by adfrancis** [comment]
You mostly nailed the features I was looking for! As a long time user of GNS3,
EVE-NG, and VIRL (CML), I live and die by my lab environment and very much look
forward to these features most:
1. Client-less (html5 web UI)
2. Scalable (supports clustering)
3. Permit customized images
4. Multiple simultaneous constantly running labs
5. No node limits

Additionally I'd love to have:
6. Expandable storage (mounting of NFS, SMB shares to store node template's
and images)
7. Multi user with RBAC
8. Supports load balancers

**Comment by adfrancis** [comment]
Will 3.0 allow labs to continue =running in the background upon exit of the webui?

**Comment by grossmj** [comment]
> Will 3.0 allow labs to continue =running in the background upon exit of the webui?

You should already be able to do it by configuring the project to keep running in the background.

![Screenshot from 2023-12-17 18-18-20](https://github.com/GNS3/gns3-gui/assets/4752614/8bfeebee-ef5e-46c0-bee4-25ab5abb7f89)

**Comment by rhysperry111** [comment, marked as answer]
I'd say about now is probably a good time ;)

**Reply by grossmj** [reply]
We have release v3.0
